sdk/python/feast/feature_store.py

```diff
@@ -488,16 +510,25 @@ def _make_inferences(
             [view.batch_source for view in views_to_update], self.config
         )
 
+        update_data_sources_with_inferred_event_timestamp_col(
+            [view.batch_source for view in sfvs_to_update], self.config
+        )
+
         # New feature views may reference previously applied entities.
         entities = self._list_entities()
         update_feature_views_with_inferred_features_and_entities(
             views_to_update, entities + entities_to_update, self.config
         )
+        update_feature_views_with_inferred_features_and_entities(
```

Comment:
this inference will break for SFVs as it relies on the underlying batch source; instead we should be doing something similar to ODFV schema inference

I think we can do that in a future PR; for now maybe don't run this inference here, and just add a warning or something in SFV if a schema isn't specified?

Reply:
Added as a future pr, also added warning